src/index.html

```diff
@@ -6,7 +6,40 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <main class="calender calender--start-day--tue calender--length--28">
+      <div class="day"></div>
```

Comment:
Because of BEM, you also need to name main as page's element, and then div as main's element. And in theory you need an empty line between the divs.